packages/dx-vue-grid-bootstrap4/src/templates/column-chooser/overlay.test.js

```diff
@@ -0,0 +1,23 @@
+import { shallow } from '@vue/test-utils';
+import { Overlay } from './overlay';
+
+describe('Overlay', () => {
+  it('should pass default slot content', () => {
+    const wrapper = shallow({
+      render() {
+        return (
+          <Overlay
+            target={{
+              getBoundingClientRect: () => ({}),
```

Comment:
Why do you need to define this method here?

Reply:
Popover requires it.